### DISCUSSION TITLE
Unhandled case 001

### DISCUSSION THREAD

**Discussion by sanaper-cloud** [hernanchavezthielemann/GRO2LAM#54] [Q&A] [answered]
Writing Lammps data file... 
####-------  ATOM STYLE < FULL >  --------#### 

==============  Still in BETA here  ==============

Sidemols: 4600

** New side molecule : SOL 1st atom : OW **
hop! SOL
here 0

** New side molecule : ETH 1st atom : C **
hop! ETH
here 0
**** Unhandled case 001

**Comment by hernanchavezthielemann** [comment]
Hello @sanaper-cloud,

I have no clue what are you trying to achieve.
Could you tell me how to reproduce the issue?? or upload a working example maybe?
Otherwise I cannot do a thing.

Best regards,
Hernan

**Comment by sanaper-cloud** [comment]
I need lammps file by converting .gro and .top file.
I am using two types of molecules SOL and ETH

![problem](https://user-images.githubusercontent.com/78227322/170636350-4fc2e620-b972-4515-a38a-2857e1d88712.png)

**Comment by hernanchavezthielemann** [comment, marked as answer]
Hello @sanaper-cloud,

I think there could be some naming mismatch between some molecules (tag), and molecule definition.
But I am not sure, since I don't know what is in your files.
You can check the examples folder that comes with GRO2LAM.

Best,
Hernan
